src/accelerate/utils/modeling.py

```diff
@@ -1214,6 +1244,12 @@ def infer_auto_device_map(
                         modules_to_treat.pop(tied_module_index)
                     device_map[tied_module_name] = devices[current_device]
 
+                if not offload_buffers:
```

Comment:
We need to add an extra condition to make sure that we are indeed passing a `nn.Module` in `compute_module_total_buffer_size` since `module` can also be a `nn.Parameter`. This is why the test was failing.

Reply:
fixed